src/services/firebase/Authentication.js

```diff
@@ -33,3 +33,25 @@ export function sendVerificationEmail (user) {
     .then(() => console.log('Email de verificação enviado'))
     .catch(() => console.error('Error ao enviar email de verificação'))
 }
+
+export const saveProfile = (profile) => {
+  var currentUser = Firebase.auth().currentUser
+  var pathsToUpdate = {}
+  pathsToUpdate[`profiles/${currentUser.uid}`] = profile
+
+  return new Promise((resolve, reject) => {
+    Firebase
```

Comment:
I thought about that but I wasn't sure because we use the Firebase.auth as well to get the current user, but now that you mention it I think it would make much more sense to move it, I'll do it in the next PR